### ISSUE TITLE
Change order of changes in release template

### ISSUE THREAD

**Issue by bidoubiwa** [meilisearch/integration-guides#148]
For the moment the release template generates the changes in the  following order: 

- Changes
- Breaking Changes

For example it would generate the following (I know some PR's should not be there haha):

```
## Changes

* Fix typo (#436) @LorenzoSapora
* Extend checkArguments to cover all arguments (#446) @ColinFrick
* Remove !important from scss (#452) @ColinFrick
* Automatic dark/light mode toggle, force dark mode (#453) @ColinFrick
* [Security] Bump ansi-regex from 5.0.0 to 5.0.1 (#454) 

## Breaking changes ⚠️

* Migrate to dart-sass, update divisions (#447) @ColinFrick
```

The problem is that if the body is too long, the content is cropped. Because `breaking changes` are after `changes` it is those changes that are cropped. 

Example: 

<img width="1039" alt="Screenshot 2021-10-21 at 18 04 50" src="https://user-images.githubusercontent.com/33010418/138315465-94f1f989-06f1-4248-9630-5f0b0e7d87cf.png">

Which is unfortunate as users have to click on `read more` to see the changes. 

I think the breaking changes should always be showcased and thus we should add them before `changes`

**Comment by curquiza** [comment]
Should be fixed by #152: the "breaking change" section will be now at the top of the release draft since there is no more "Change" default section

**Comment by brunoocasali** [comment]
It seems to be fixed 🎉  Thanks Clem!
